engine/bench_test.go

```diff
@@ -42,7 +42,7 @@ func BenchmarkChunkDecoding(b *testing.B) {
 			ss := querier.Select(false, nil, matcher)
 			series := make([]chunkenc.Iterator, 0)
 			for ss.Next() {
-				series = append(series, ss.At().Iterator())
+				series = append(series, ss.At().Iterator(nil))
```

Comment:
Is there a way we can take advantage of reusing these iterators? Should we open an issue to investigate?

Reply:
I guess it's something worth looking into. I will create an issue and link details from Prom repo.